android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -1236,6 +1281,40 @@ public void run() {
     }, delay);
   }
 
+  protected void showHelpBubble() {
+    String hintText = context.getString(R.string.help_bubble_text);
+
+    // signalHelpBubbleDismissal - defined in android-host.js, gives a helpBubbleDismissed signal.
```

Comment:
showHelpBubble() should NOOP if keyboardType == KeyboardType.KEYBOARD_TYPE_SYSTEM

See from stable-15.0
https://github.com/keymanapp/keyman/blob/99680a2569123033c2ee34e1cbd30a9c81d16f75/android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java#L1277-L1279